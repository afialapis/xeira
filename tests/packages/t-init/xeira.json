{
  "product": "package",
  "target": "both",
  "source_index": "./src/index.js",
  "react": true,
  "monorepo": false,
  "linter": "eslint",
  "transpile_folder": "./lib",
  "transpiler": "babel",
  "minifier": "uglify",
  "bundle_folder": "./dist",
  "bundler": "rollup",
  "test_folder": "./test",
  "demo_mode": "auto",
  "demo_demoer": "rollup"
}
