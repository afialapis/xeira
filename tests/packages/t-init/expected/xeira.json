{
  "product": "package",
  "target": "both",
  "source_index": "./src/index.js",
  "linter": "eslint",
  "transpile_folder": "./lib",
  "transpiler": "babel",
  "minifier": "uglify",
  "bundle_folder": "./dist",
  "bundler": "rollup",
  "test_folder": "./test",
  "react": true,
  "monorepo": false
}
